Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -857,33 +874,50 @@ void NetworkStorageManager::setItem(IPC::Connection& connection, StorageAreaIden
     ASSERT(!RunLoop::isMain());
 
     bool hasQuotaError = false;
-    if (auto storageArea = m_storageAreaRegistry->getStorageArea(identifier)) {
-        auto result = storageArea->setItem(connection.uniqueID(), implIdentifier, String { key }, WTFMove(value), WTFMove(urlString));
-        if (!result)
-            hasQuotaError = (result.error() == StorageError::QuotaExceeded);
-    }
+    auto storageArea = m_storageAreaRegistry->getStorageArea(identifier);
+    if (!storageArea)
+        return completionHandler(hasQuotaError);
 
+    auto result = storageArea->setItem(connection.uniqueID(), implIdentifier, String { key }, WTFMove(value), WTFMove(urlString));
```

Comment:
nit: Why isn't this using `WTFMove(key)` ?

Reply:
I don't remember; but should be fine to WTFMove...